src/pages/gateway/release-notes.md

```diff
@@ -14,6 +14,16 @@ keywords:
 
 The following sections indicate when updates were made to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading versions.
 
+# December 14, 2023
```

Comment:
Is this date correct?

Reply:
@keharper no, but the date is still TBD. Thanks for the reminder. I'll update before publishing.